deploy/olm-catalog/ibm-auditlogging-operator/3.7.0/ibm-auditlogging-operator.v3.7.0.clusterserviceversion.yaml

```diff
@@ -373,19 +472,17 @@ spec:
                 - ibm-auditlogging-operator
                 env:
                 - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
I'll need to revert back to `metadata.annotations['olm.targetNamespaces']` once I test with odlm deploying the operator as `installMode: cluster`

Reply:
Installing to `openshift-operators` may break us in an upgrade scenario